src/api/api.types.tsx

```diff
@@ -217,3 +217,23 @@ export interface Item
   usage_status: string;
   properties: Property[];
 }
+
+// ------------------------------------ ATTACHMENTS ------------------------------------------------
+
+export interface AttachmentsPostMetadata {
+  entity_id: string;
+  file_name: string;
+  title?: string | null;
+  description?: string | null;
+}
+
+export interface UploadInfo {
+  url: string;
+  fields: Record<string, string>;
+}
+export interface AttachmentsPostMetadataResponse
```

Comment:
```suggestion
export interface AttachmentPostResponse
```
depending on the above